### DISCUSSION TITLE
[Proposal] Default value for returned tuples

### DISCUSSION THREAD

**Discussion by aleksandrsmyk** [dotnet/csharplang#6378] [Language Ideas]
It will be flexible to set default values for returned tuples 

Now, I have to set isOK parameter always:

_IEnumerable<(string p1, bool isOK)> SimpleFunction()
{
   yield return new ("value1", true);
   yield return new ("value2", true);
   yield return new ("value1", false);
}_

But it will be flexible to have an ability to write:

IEnumerable<(string p1, **bool isOK = true**)> SimpleFunction()
{
   yield return new ("value1");
   yield return new ("value2");
   yield return new ("value1", **isOK:false**);
}

**Comment by ufcpp** [comment]
```cs
IEnumerable<(string p1, bool isOK)> SimpleFunction()
{
    static (string, bool) r(string p1, bool isOK = true) => (p1, isOK);
    yield return r("value1");
    yield return r("value2");
    yield return r("value1", isOK: false);
}
```

**Reply by aleksandrsmyk** [reply]
Its acceptable decision, but it makes code a little bit complicated in reading.
